homeassistant/components/fritzbox_callmonitor/sensor.py

```diff
@@ -82,6 +82,9 @@ class FritzBoxCallSensor(SensorEntity):
     """Implementation of a Fritz!Box call monitor."""
 
     _attr_icon = ICON_PHONE
+    _attr_translation_key = DOMAIN
+    _attr_device_class = SensorDeviceClass.ENUM
+    _attr_options = [cls.value for cls in CallState]
```

Comment:
```suggestion
    _attr_options = list(CallState)
```
`CallState` is a `StrEnum`